src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+@import "../utils/variables";
+
+.page {
+
+  &__calendar {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gapX;
+    width: 7 * $sizeBlock + 7 * $gapX;
+    margin: 0 auto;
```

Comment:
10. [STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover.